### PR TITLE
Initial commit for game logic

### PR DESCRIPTION
**Pull request by Nipunwahi**
Added initial commit for game logic

### CODE REVIEW

File: src/Game.hs [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+data Value = Two | Three | Four | Five | Six | Seven | Eight | Nine | Ten | Joker | Queen | King | Ace deriving (Ord,Eq,Show,Enum,Read)
+data Card = Card {suit:: Suit, value::Value} deriving (Eq,Show,Read)
+
+
+class Named a where
+    name :: a -> String
+
+
+data Player = Player {_nameP::String, cards::[Card]} deriving (Eq,Show,Read)
+data Team = Team {_nameT::String, points::Int, players::(String,String)} deriving (Eq,Show,Read)
```
Comment (anishkoulgi) [inline src/Game.hs:23]: Should players be tuple of `Player` rather than string?

**Review by anishkoulgi — APPROVED** [review]
RLGTM

File: src/Game.hs (hunk 1 above)
Comment (Nipunwahi) [reply src/Game.hs:23]: I did that but we have to update the cards at two places then after i change the state , so i changed that to strings to reduce changes multiple places

Comment (anishkoulgi) [reply src/Game.hs:23]: Got it, looks good 👍

**Pull request merged**